distro/assembly/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM jboss/wildfly
+
+RUN $JBOSS_HOME/bin/add-user.sh admin artificer1! --silent
```

Comment:
@jorgemoralespou, is this line really needed?  The ant installation's `-Dejb-jms.password=artificer1!` creates an `artificer` JBoss user under the hood.  We're no longer using an `admin` user for anything.